package.json

```diff
@@ -71,7 +72,6 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^7.4.0",
```

Comment:
i don't want to remove this from package.json. can you npm install it and resubmit?